OxfordBodleian/Juel-Jensen/BDLaethd19.xml

```diff
@@ -456,26 +512,958 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                <q xml:lang="gez">ሚካኤል፡ መልአክ።</q>
+                                <q xml:lang="gez">ሥዕለ፡ ጴጥሮስ፡ ወጳውሎስ።</q>
+                                <q xml:lang="gez">መራኁት።</q>
+                                <q xml:lang="gez">ዮሐንስ፡ መጥምቅ፡</q>
+                                <q xml:lang="gez">እንድርያስ፡ ሐዋርያ፡</q>
+                                <q xml:lang="gez">ሰአሉ፡ ለነ።</q>
+                                <q xml:lang="gez">ያዕቆብ<unclear>፡</unclear> እኁሁ፡ ለእግዚእነ፡</q>
+                                <q xml:lang="gez">ማርቆስ። ። ።</q>
+                                <q xml:lang="gez">በርተሎሜዎስ፡</q>
+                                <q xml:lang="gez">ማቴዎስ፡</q>
+                                <q xml:lang="gez">ቶማስ፡</q>
+                                <q xml:lang="gez">ፊልጶስ፡</q>
+                            </decoNote>
+                            
+                        </decoDesc>
+                        
+                        <additions>
+                            <list>
+
+                                
+                                <item xml:id="e3">
+                                    There are frequent line takeups and line takedowns (<locus target="#1v #7r #8r"/>), occasional erasures 
+                                    (<locus target="#15v #16v #17v #22r #35v #70v #89r"/>), and interlinear additions (<locus target="#47r #54v"/>). 
+                                    In the upper margin of every page where a Psalm or Canticle begins, there is a short phrase taken (one to three
+                                    words) taken from the respective Psalm or Canticle. The line number of each Psalm, Canticle, or part of the
+                                    Song of Songs has been provided in the margin, placed in a decorative box.
+                                </item>                                
+                             </list>
+                        </additions>
+                        
+                    </physDesc>
+
+
+                    </msPart>
+                    
+                    <msPart xml:id="p2">
+                        <msIdentifier>
+                            <idno/>
+                        </msIdentifier>
+                    
+                    <msContents>
+                        <summary/>
+                        
+                        <msItem xml:id="p2_i1">
+                            <locus from="148ra" to="149vb"/>
+                            <title type="incomplete" ref="LIT2715John"/>
+                            <note>The text is incomplete and contains Jn 20:20–21:21, ending abruptly in the middle of the last verse.</note>
+                            <incipit xml:lang="gez">
+                                <locus target="#148ra"/>
+                                <lb/>እገሪሁ፡ ወገቦሁ<hi rend="rubric" rendition="#partialRubric">፨</hi> ወ
```

Comment:
typically, `<lb/> ` has not been used in two-column layout. We might discuss where we want a line break to be printed

Reply:
Does it cause a problem to have it encoded like this? I was thinking that it won't be rendered

Reply:
Let's keep it for now, it would be a pity to lose the information

Reply:
It would not be the end of the world, but if it doesn't hurt, it can be there :)